Sources/Network/HTTPS/Utils/URL+Prefix.swift

```diff
@@ -6,12 +6,13 @@ import Foundation
 
 extension URL {
     static func prefix(_ url: URL, pathComponents: [String]) -> URL? {
-        let prunedComponents = pathComponents.map({ $0.hasPrefix("/") ? String($0.dropFirst()) : $0})
+        let prunedComponents = pathComponents.map({
+                $0.hasPrefix("/") ? String($0.dropFirst()) : $0
```

Comment:
Single-indent block body relative to wrapping { and }... Even though we double-indent wrapped params, since this is a block body, it should be single-indented here I believe:

```suggestion
            $0.hasPrefix("/") ? String($0.dropFirst()) : $0

```

Reply:
subjective indentation is ok